### DISCUSSION TITLE
我应该向cslab上传什么？

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#150] [Q&A] [answered]
如题，如果只上传.cpp，.h和makefile,那么在windows的vs2019上无法运行,如国将sln等也上传,z则在linux下也无法运行，我该怎么办

**Comment by binbin00** [comment, marked as answer]
基础功能版本在其中一个环境能运行即可，下周提交的版本只要求在linux下运行。
